supertokens_python/recipe/dashboard/api/userdetails/user_sessions_get.py

```diff
@@ -23,7 +23,10 @@ async def handle_sessions_get(
     if user_id is None:
         raise_bad_input_exception("Missing required parameter 'userId'")
 
-    session_handles = await get_all_session_handles_for_user(user_id, user_context)
+    # TODO: Pass tenant id here
+    session_handles = await get_all_session_handles_for_user(
+        user_id, "pass-tenant-id", user_context
```

Comment:
this won't have tenant id passed to it. You need to pass `None` to it instead so that it gets all session's for the user across all the tenants. See node PR

Reply:
Updated.